Curve-finance-suite/src/agents/kill.me.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from 'forta-agent';
+
+import Web3 from 'web3';
+import abi from '../utils/stable.swap.abi';
+
+// @ts-ignore
+import abiDecoder from 'abi-decoder';
+abiDecoder.addABI(abi);
+
+export const web3 = new Web3();
+
+export const killme = {
+  name: 'kill_me',
+  outputs: [],
+  inputs: [],
+  stateMutability: 'nonpayable',
+  type: 'function',
+  gas: 38058,
+};
+
+const createFinding = (alertId: string): Finding => {
+  return Finding.fromObject({
+    name: 'Kill Me funciton called',
+    description: 'Kill Me funciton called on pool',
```

Comment:
on what pool? Is the address field in metadata the pool? Can you add the pool in the metadata? Can you add any metadata to make this finding more informative?